packages/components/src/styles/global.tsx

```diff
@@ -0,0 +1,6 @@
+import React, { ReactElement } from "react";
+import { GlobalStyles } from "twin.macro";
+
+const EDSGlobalStyles = (): ReactElement => <GlobalStyles />;
+
+export default EDSGlobalStyles;
```

Comment:
_Why remove the Fonts export?_
- traject already imports Arimo
- I had trouble including the google import inside `createGlobalStyles` (https://github.com/styled-components/styled-components/pull/2997)

I vaguely feel like it's better to let the application decide how to handle importing the required font anyways? but it is an extra setup step

Reply:
Agreed that the application can/should import the font in the best way for it. Different apps may need to use different fonts, as well.